app/scripts/components/analysis/page-hero-media.tsx

```diff
@@ -74,18 +76,16 @@ function PageHeroMedia(props: PageHeroMediaProps) {
 
     const aoiSource = mapRef.current.getSource('aoi');
 
-    // Quick copy.
-    const featureInverse = JSON.parse(
-      JSON.stringify(feature)
-    ) as Feature<MultiPolygon>;
+    // Convert to multipolygon to use the inverse shading trick.
+    const aoiInverse = combineFeatureCollection(aoi);
     // Add a full polygon to reverse the feature.
-    featureInverse.geometry.coordinates[0].unshift(WORLD_POLYGON);
+    aoiInverse.geometry.coordinates[0].unshift(WORLD_POLYGON);
```

Comment:
I'd follow eslint advice here:
```suggestion
    aoiInverse.geometry.coordinates[0] = [WORLD_POLYGON, ...aoiInverse.geometry.coordinates[0]];
```

Reply:
We're already mutating the object, my reasoning is that this way just makes it clearer. Nevertheless, no strong opinion on it.

Reply:
I have a vaguely strong opinion that this is easier to read (I mean, `unshift` really is a badly named method)

Reply:
yes it is! terrible really :p